From 129f23ce758620fade812baab811379ce8454048 Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Fri, 27 Jan 2017 11:44:41 +0100
Subject: [PATCH] test-lock: disable the rwlock test

It hangs indefinitely if the system rwlock implementation does not
prevent writer starvation (and glibc does not implement it).

Bug: http://www.mail-archive.com/bug-gnulib@gnu.org/msg33017.html
---
 tests/test-lock.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-lock.c b/tests/test-lock.c
index a992f64..fd9c014 100644
--- a/tests/test-lock.c
+++ b/tests/test-lock.c
@@ -42,7 +42,7 @@
    Uncomment some of these, to verify that all tests crash if no locking
    is enabled.  */
 #define DO_TEST_LOCK 1
-#define DO_TEST_RWLOCK 1
+#define DO_TEST_RWLOCK 0
 #define DO_TEST_RECURSIVE_LOCK 1
 #define DO_TEST_ONCE 1
 
-- 
2.7.4

